whois-query/src/test/java/net/ripe/db/whois/query/pipeline/ConnectionPerIpLimitHandlerTest.java

```diff
@@ -46,6 +46,7 @@
 
     @Before
     public void setUp() {
+        System.setProperty("instance.name", "10.0.0.0");
```

Comment:
Make sure to use `System.clearProperty("instance.name")` in an `@After` method also.